stories/components/mdx/BlockQuote.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react'
+import BlockQuote from '../../../components/mdx/BlockQuote'
+import Layout from '../../../components/mdx/Layout'
+import { MockedProvider } from '@apollo/client/testing'
+import GET_APP from '../../../graphql/queries/getApp'
+import dummyLessonData from '../../../__dummy__/lessonData'
+import dummySessionData from '../../../__dummy__/sessionData'
+import { withTestRouter } from '../../../__tests__/utils/withTestRouter'
+
+export default {
+  component: BlockQuote,
+  title: 'mdx/BlockQuote'
+}
+const mocks = [
+  {
+    request: { query: GET_APP },
+    result: {
+      data: {
+        lessons: dummyLessonData,
+        session: dummySessionData,
+        alerts: []
+      }
+    }
+  }
+]
+export const Quote: React.FC = () => {
+  return withTestRouter(
+    <MockedProvider mocks={mocks} addTypename={false}>
+      <Layout>
```

Comment:
Is the `<Layout>` component needed for this story? Without it you could make it more simple without the mocks and the mocked router.

Reply:
Blockquote is a grey background and lightbulb image. Storybook uses some grayish background too and so this component looks weird without proper white layout.